From 05b025edf4aecf19634e01b0974126e53a927a50 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mateusz=20Miku=C5=82a?= <mati865@gmail.com>
Date: Sun, 1 Aug 2021 23:26:25 +0300
Subject: [PATCH] [LLD][MinGW] Accept joined format for --stack

Postgresql uses `--stack=` in its Makefile.

Downstream issue: https://github.com/msys2/MINGW-packages/pull/9167

Reviewed By: mstorsjo

Differential Revision: https://reviews.llvm.org/D107237
---
 MinGW/Options.td       | 2 +-
 test/MinGW/driver.test | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/MinGW/Options.td b/MinGW/Options.td
index e1a505240cb82..7b250614e434b 100644
--- a/MinGW/Options.td
+++ b/MinGW/Options.td
@@ -85,7 +85,7 @@ defm output_def: Eq<"output-def", "Output def file">;
 defm section_alignment: Eq<"section-alignment", "Set section alignment">;
 def shared: F<"shared">, HelpText<"Build a shared object">;
 defm subs: Eq<"subsystem", "Specify subsystem">;
-def stack: S<"stack">;
+defm stack: Eq<"stack", "Set size of the initial stack">;
 def strip_all: F<"strip-all">,
     HelpText<"Omit all symbol information from the output binary">;
 def strip_debug: F<"strip-debug">,
